website_chd_product_configurator/static/src/js/website_chd_product_configurator.js

```diff
@@ -0,0 +1,121 @@
+$(document).ready(function () {
+
+	$('.type_selection').on('change', function ()
+    {
+    	openerp.jsonRpc('/chd_init/getch/', 'call',
+    		 {'type_id': this.value,}).then(function (data)
+    				 {
+    			            $('#fini_select_id').empty();
+    			  			$('#fini_select_id').append("<option></option>");
+    			  			var data_js = eval(data);
```

Comment:
you don't need to jsondump/eval your data for a json controller call. The framework does that for you, just return the python structure and it will come out as something you can use in js. But with the above, you don't need the controller anyway, so this line can go away